### ISSUE TITLE
Spectrometer 3.0 graphics

### ISSUE THREAD

**Issue by mathewlippincott** [publiclab/design#14]
We need to figure out what graphics to place on the new spectrometer. I'm going to propose that we only print on one of the two pieces and leave the top unprinted. 
![spectrometer internal](https://cloud.githubusercontent.com/assets/1203280/5463274/183d4556-8530-11e4-8829-7f218bc9344a.png)

I want to print the relationship of parts and references to @cfastie and other's notes on one big side, and titling on the other.
![screen shot 2014-12-16 at 2 25 35 pm](https://cloud.githubusercontent.com/assets/1203280/5463197/7f53e46c-852f-11e4-8087-bdd13bbe80a4.png)

I want to explain the /n/##### notes on the bottom, maybe a list of materials or prototyping resources.

**Comment by mathewlippincott** [comment]
this is getting there.   need to add the notes in.
![screen shot 2014-12-16 at 5 49 33 pm](https://cloud.githubusercontent.com/assets/1203280/5465457/f2fa79ea-854c-11e4-8645-c80bac718d77.png)

**Comment by jywarren** [comment]
Can we print a light cm grid on there too? Good for modding and such.
Looks great.

**Comment by mathewlippincott** [comment]
![screen shot 2014-12-17 at 8 38 35 pm](https://cloud.githubusercontent.com/assets/1203280/5483731/0503375e-862d-11e4-8c3a-92406aee108b.png)
![screen shot 2014-12-17 at 8 38 19 pm](https://cloud.githubusercontent.com/assets/1203280/5483732/09f4b8be-862d-11e4-824a-1890d5604c50.png)
![screen shot 2014-12-17 at 8 38 27 pm](https://cloud.githubusercontent.com/assets/1203280/5483734/0cd05872-862d-11e4-9fa2-8642a68df3e7.png)

**Comment by mathewlippincott** [comment]
![screen shot 2014-12-17 at 8 39 57 pm](https://cloud.githubusercontent.com/assets/1203280/5483736/1729cc7c-862d-11e4-9d43-29ef0c8ee632.png)

**Comment by Nataliegurl** [comment]
Sticker label graphics etc. @mathewlippincott

**Comment by mathewlippincott** [comment]
box sticker in dropbox:
plots/design/12-16-14-spectrometer3.0 gfx/box label
![screen shot 2015-01-22 at 5 10 19 pm](https://cloud.githubusercontent.com/assets/1203280/5868100/d48a2666-a259-11e4-8e25-85de17922213.png)

**Comment by Nataliegurl** [comment]
Forwarded to Charles for run

**Comment by Nataliegurl** [comment]
Can we update the bundle picture?
![oldbundle](https://cloud.githubusercontent.com/assets/7808557/6253261/c8278d5e-b754-11e4-8082-5a45e4352718.png)

**Comment by jywarren** [comment]
yes please; could I just do a single ensemble photo? It needs the 3.0 spec,
right?

**Comment by Nataliegurl** [comment]
Yep, that's it

## 

Natalie Mayorga

_Kits Initiative ManagerPublic Lab412 NW Couch, Ste. 401Portland, OR 97209_

_T: 503.902.5874_

_F: 504.324.0401_

_Natalie@publiclab.org Natalie@publiclab.org_

_www.publiclab.org http://www.publiclab.org_

_FB: www.facebook.com/PublicLab http://www.facebook.com/PublicLab_

_Twitter: @PublicLab, @natcat5050_
_Instagram: public_lab_

**Comment by Nataliegurl** [comment]
also the picture on this page needs to be updated,
http://publiclab.org/wiki/spectrometer
I don't know how to save a picture as an address.

## 

Natalie Mayorga

_Kits Initiative ManagerPublic Lab412 NW Couch, Ste. 401Portland, OR 97209_

_T: 503.902.5874_

_F: 504.324.0401_

_Natalie@publiclab.org Natalie@publiclab.org_

_www.publiclab.org http://www.publiclab.org_

_FB: www.facebook.com/PublicLab http://www.facebook.com/PublicLab_

_Twitter: @PublicLab, @natcat5050_
_Instagram: public_lab_
